### ISSUE TITLE
What changed in v1.0.1? Not working

### ISSUE THREAD

**Issue by Snuffy2** [chazzu/hass-animated-scenes#20]
I've been using your integration and have really liked it. I updated to v1.0.1 from an earlier version (`f1d6151` I believe) by HACS and now it isn't working. It appears you moved from setting the animations in a switch to doing it in a service call now? Any documentation on the new syntax? The readme still shows the switch setup.

Happy to update my animations, I just want to confirm I'm doing it correctly as I've been using the switches heavily and moving to service calls will involve significant modification to my current setup for my lights.

Also, it would have been helpful if this was noted as a breaking change in the release notes of the update so we could be aware/prepared. Thanks :-)

**Comment by Snuffy2** [comment]
I had to revert `animated_scenes` to the prior version in my Production HA as it was preventing restarting HA. I installed v1.0.1 in my Dev HA and added my `animated_scenes` switches.

Log:
```
2023-03-30 09:40:53.822 WARNING (SyncWorker_3) [homeassistant.loader] We found a custom integration animated_scenes which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-03-30 09:41:14.016 ERROR (MainThread) [homeassistant.config] Invalid config for [switch.animated_scenes]: [brightness] is an invalid option for [switch.animated_scenes]. Check: switch.animated_scenes->brightness. (See /config/switch.yaml, line 1). Please check the docs at https://github.com/chazzu/hass-animated-scenes#readme
2023-03-30 09:41:14.061 ERROR (MainThread) [homeassistant.config] Invalid config for [switch.animated_scenes]: [brightness] is an invalid option for [switch.animated_scenes]. Check: switch.animated_scenes->brightness. (See /config/switch.yaml, line 27). Please check the docs at https://github.com/chazzu/hass-animated-scenes#readme
2023-03-30 09:41:14.069 ERROR (MainThread) [homeassistant.config] Invalid config for [switch.animated_scenes]: [brightness] is an invalid option for [switch.animated_scenes]. Check: switch.animated_scenes->brightness. (See /config/switch.yaml, line 46). Please check the docs at https://github.com/chazzu/hass-animated-scenes#readme
2023-03-30 09:41:14.076 ERROR (MainThread) [homeassistant.config] Invalid config for [switch.animated_scenes]: [brightness] is an invalid option for [switch.animated_scenes]. Check: switch.animated_scenes->brightness. (See /config/switch.yaml, line 71). Please check the docs at https://github.com/chazzu/hass-animated-scenes#readme
2023-03-30 09:41:14.083 ERROR (MainThread) [homeassistant.config] Invalid config for [switch.animated_scenes]: [brightness] is an invalid option for [switch.animated_scenes]. Check: switch.animated_scenes->brightness. (See /config/switch.yaml, line 96). Please check the docs at https://github.com/chazzu/hass-animated-scenes#readme
2023-03-30 09:41:14.090 ERROR (MainThread) [homeassistant.config] Invalid config for [switch.animated_scenes]: [brightness] is an invalid option for [switch.animated_scenes]. Check: switch.animated_scenes->brightness. (See /config/switch.yaml, line 121). Please check the docs at https://github.com/chazzu/hass-animated-scenes#readme
2023-03-30 09:41:14.097 ERROR (MainThread) [homeassistant.config] Invalid config for [switch.animated_scenes]: [brightness] is an invalid option for [switch.animated_scenes]. Check: switch.animated_scenes->brightness. (See /config/switch.yaml, line 144). Please check the docs at https://github.com/chazzu/hass-animated-scenes#readme
2023-03-30 09:41:14.104 ERROR (MainThread) [homeassistant.config] Invalid config for [switch.animated_scenes]: [brightness] is an invalid option for [switch.animated_scenes]. Check: switch.animated_scenes->brightness. (See /config/switch.yaml, line 179). Please check the docs at https://github.com/chazzu/hass-animated-scenes#readme
```

When trying to restart:
```
Invalid config for [switch.animated_scenes]: [brightness] is an invalid option for [switch.animated_scenes]. Check: switch.animated_scenes->brightness. (See ?, line ?). 
Invalid config for [switch.animated_scenes]: [brightness] is an invalid option for [switch.animated_scenes]. Check: switch.animated_scenes->brightness. (See ?, line ?). 
Invalid config for [switch.animated_scenes]: [brightness] is an invalid option for [switch.animated_scenes]. Check: switch.animated_scenes->brightness. (See ?, line ?). 
Invalid config for [switch.animated_scenes]: [brightness] is an invalid option for [switch.animated_scenes]. Check: switch.animated_scenes->brightness. (See ?, line ?). 
Invalid config for [switch.animated_scenes]: [brightness] is an invalid option for [switch.animated_scenes]. Check: switch.animated_scenes->brightness. (See ?, line ?). 
Invalid config for [switch.animated_scenes]: [brightness] is an invalid option for [switch.animated_scenes]. Check: switch.animated_scenes->brightness. (See ?, line ?). 
Invalid config for [switch.animated_scenes]: [brightness] is an invalid option for [switch.animated_scenes]. Check: switch.animated_scenes->brightness. (See ?, line ?). 
Invalid config for [switch.animated_scenes]: [brightness] is an invalid option for [switch.animated_scenes]. Check: switch.animated_scenes->brightness. (See ?, line ?). 
```

A few of my previously working switches:
```yaml
- platform: animated_scenes
  name: "Front Porch Christmas Lights"
  ignore_off: False
  lights:
    - light.lifx_front_porch_light_bulb_1
    - light.lifx_front_porch_light_bulb_2
    - light.lifx_front_porch_light_bulb_3
  brightness: 255
  colors:
    - color_type: rgb_color
      color: [ 255, 0, 0 ]
    - color_type: rgb_color
      color: [ 240, 255, 254 ]
    - color_type: rgb_color
      color: [ 255, 98, 71 ]
    - color_type: rgb_color
      color: [ 255, 214, 0 ]
      weight: 5
    - color_type: rgb_color
      color: [ 125, 255, 0 ]
    - color_type: rgb_color
      color: [ 0, 255, 0 ]
  transition: [2, 6]
  change_frequency: [2, 6]
  change_amount: [1, 3]

- platform: animated_scenes
  name: Front Porch Patriotic Lights
  ignore_off: False
  lights:
    - light.lifx_front_porch_light_bulb_1
    - light.lifx_front_porch_light_bulb_2
    - light.lifx_front_porch_light_bulb_3
  brightness: 255
  colors:
    - color_type: rgb_color
      color: [ 255, 0, 0 ]
    - color_type: rgb_color
      color: [ 255, 255, 255 ]
    - color_type: rgb_color
      color: [ 0, 0, 255 ]
  transition: [2, 6]
  change_frequency: [2, 6]
  change_amount: all

- platform: animated_scenes
  name: Front Porch Valentines Lights
  ignore_off: False
  lights:
    - light.lifx_front_porch_light_bulb_1
    - light.lifx_front_porch_light_bulb_2
    - light.lifx_front_porch_light_bulb_3
  brightness: [100, 255]
  colors:
    - color_type: rgb_color
      color: [ 255, 0, 0 ]
    - color_type: rgb_color
      color: [ 255, 255, 255 ]
    - color_type: rgb_color
      color: [ 220, 20, 60 ]
    - color_type: rgb_color
      color: [ 255, 250, 250 ]
    - color_type: rgb_color
      color: [ 255, 20, 147 ]
    - color_type: rgb_color
      color: [ 255, 105, 180 ]
  transition: [2, 6]
  change_frequency: [2, 6]
  change_amount: [1, 3]

- platform: animated_scenes
  name: Front Porch NYE Lights
  ignore_off: False
  lights:
    - light.lifx_front_porch_light_bulb_1
    - light.lifx_front_porch_light_bulb_2
    - light.lifx_front_porch_light_bulb_3
  brightness: 255
  colors:
    - color_type: rgb_color
      color: [ 0,0,128 ]
    - color_type: rgb_color
      color: [ 64,224,208 ]
    - color_type: rgb_color
      color: [ 255, 255, 255 ]
    - color_type: rgb_color
      color: [ 192,192,192 ]
    - color_type: rgb_color
      color: [ 72,61,139 ]
    - color_type: rgb_color
      color: [ 0,0,255 ]
  transition: [2, 6]
  change_frequency: [2, 6]
  change_amount: all

- platform: animated_scenes
  name: Front Porch Easter Lights
  ignore_off: False
  lights:
    - light.lifx_front_porch_light_bulb_1
    - light.lifx_front_porch_light_bulb_2
    - light.lifx_front_porch_light_bulb_3
  brightness: [100, 255]
  colors:
    - color_type: rgb_color
      color: [ 255,255,0 ]
    - color_type: rgb_color
      color: [ 0,250,154 ]
    - color_type: rgb_color
      color: [ 64,224,208 ]
    - color_type: rgb_color
      color: [ 255,192,203 ]
    - color_type: rgb_color
      color: [ 255,250,205 ]
    - color_type: rgb_color
      color: [ 230,230,250 ]
  transition: [2, 6]
  change_frequency: [2, 6]
  change_amount: [1, 3]

- platform: animated_scenes
  name: "Front Porch Halloween Lights"
  ignore_off: False
  lights:
    - light.lifx_front_porch_light_bulb_1
    - light.lifx_front_porch_light_bulb_2
    - light.lifx_front_porch_light_bulb_3
  brightness: 255
  colors:
    - color_type: rgb_color
      color: [ 255, 195, 0 ]
    - color_type: rgb_color
      color: [ 255, 208, 114 ]
    - color_type: rgb_color
      color: [ 255, 169, 0 ]
    - color_type: rgb_color
      color: [ 255, 54, 6 ]
    - color_type: rgb_color
      color: [ 255, 140, 0 ]
  transition: [2, 6]
  change_frequency: [2, 6]
  change_amount: [1, 3]

- platform: animated_scenes
  name: Alexs Bedroom Rainbow Sequence
  ignore_off: False
  restore: True
  restore_power: True
  animate_brightness: True
  animate_color: True
  lights:
    - light.localtuya_alexs_bedroom_light_bulb_1
    - light.localtuya_alexs_bedroom_light_bulb_2
    - light.localtuya_alexs_bedroom_light_bulb_3
    - light.localtuya_alexs_bedroom_light_bulb_4
  colors:
    - color_type: rgb_color
      color: [ 255, 0, 0 ]
    - color_type: rgb_color
      color: [ 255, 153, 0 ]
    - color_type: rgb_color
      color: [ 255, 255, 0 ]
    - color_type: rgb_color
      color: [ 0, 255, 0 ]
    - color_type: rgb_color
      color: [ 0, 0, 255 ]
    - color_type: rgb_color
      color: [ 0, 255, 255 ]
    - color_type: rgb_color
      color: [ 153, 51, 255 ]
    - color_type: rgb_color
      color: [ 255, 153, 255 ]
  transition: 1
  brightness: [128, 217]
  change_sequence: False
  change_frequency: [2, 5]
  change_amount: [1, 4]
```

**Comment by chazzu** [comment]
I have tested these and released version 1.0.2 which fixes the two issues I found. Sorry about that, this last update was fairly major in order to accommodate multiple scenes running at the same time and a new interface for updating scenes. They should now all work again if you update to 1.0.2

**Comment by Snuffy2** [comment]
Great to hear, thanks. Sorry I jumped to it being a breaking change. I'll test a little more in Dev and move it to Prod if all looks good. Thanks for the quick fix.
